strings/cas_white.txt

```diff
@@ -330,4 +330,17 @@ Being roasted so devastatingly by the AI it counts as human harm.
 A turkey made of cybernetic hate.
 A thicc securitron.
 Gluing five drones together and using them as a buttplug.
-All the exciting things happening on the station while you play Cards Against Spess.
\ No newline at end of file
+The gas giant.
+A Codex Cicatrix.
+A Living Heart with the wrong body fluid inside of it.
+Gender reassignment surgery.
```

Comment:
🤨

Reply:
/datum/surgery/gender_reassignment
	name = "gender reassignment"
	steps = list(/datum/surgery_step/incise, /datum/surgery_step/clamp_bleeders, /datum/surgery_step/reshape_genitals, /datum/surgery_step/close)
	species = list(/mob/living/carbon/human)
	possible_locs = list("groin")

Reply:
i know i was there, a bit tasteless imo

Reply:
It's CAS, that's kind of the point.

Reply:
If you REALLY don't want this card I can replace it with "surgical sexual harrasment".

Reply:
What's github gonna say when they find out

Reply:
Probably nothing because people who got PTSD from having their spessman's penis inverted against their will aren't recognized as a protected minority by github.

Reply:
You'll come to find GitHub probably cares less about context than you think.

Reply:
Well good thing this string only consists of a single compound noun with no negative connotations and a full stop.

Reply:
test

Reply:
Resolving for technical reasons.